src/lib_windows.cr

```diff
@@ -105,6 +109,40 @@ lib LibWindows
 
   @[CallConvention("X86_StdCall")]
   fun wsa_startup = WSAStartup(version : Int16, data : WSAData*) : Int32
+
+  # source https://msdn.microsoft.com/en-us/library/windows/desktop/ms724950(v=vs.85).aspx
+  struct SystemTime
+    year : Word
+    month : Word
+    day_of_week : Word
+    day : Word
+    hour : Word
+    minute : Word
+    second : Word
+    milliseconds : Word
+  end
+
+  # source https://msdn.microsoft.com/en-us/library/windows/desktop/ms725481(v=vs.85).aspx
+  struct TimeZoneInformation
+    bias : Long
+    standard_name : StaticArray(WChar, 32)
+    standard_date : SystemTime
+    standard_bias : Long
+    daylight_name : StaticArray(WChar, 32)
+    daylight_date : SystemTime
+    daylight_bias : Long
+  end
+
+  struct FileTime
+    low_date_time : DWord
+    high_date_time : DWord
+  end
```

Comment:
It would even be safe to declare it as `GetSystemTimeAsFileTime(time : UInt64*)`. The problem would be if the struct were declared as UInt32, allocated as such and then accessed by a Uint64* if it were on a architecture were accessing misaligned memory caused a crash (This causes problems on ARM, and Window can run on ARM). But it is pretty safe to declare those pairs of high/low DWords as a single UInt64 as long as the struct is allocated by you and not returned from a function.

Are you a lot against doing it?